spec.md

```diff
@@ -259,14 +259,15 @@ will need to provide this identification information to the Service Broker on
 each request. Platforms MAY support this feature, and if they do, they MUST
 adhere to the following:
 - For any OSBAPI request that is the result of an action taken by a Platform's
-  user, there MUST be an associated `X-Broker-API-Originating-Identity` header on that HTTP
-  request.
-- Any OSBAPI request that is not associated with an action from a Platform's
+  user, there MUST be an associated `X-Broker-API-Originating-Identity` header
+  on that HTTP request.
+- When a request is associated with an action from a Platform's user, the
```

Comment:
@duglin I was curious what this sentence adds to the statement above that operations by a Platform's user must include the header

Reply:
I finally remember why I did that! :-)  Its because the first bullet just said the header had to be there, but it didn't mandate that the value had to be associated with the user. I merged that thought into the previous bullet to avoid this confusion in the future.